index.js

```diff
@@ -10,27 +10,52 @@
  */
 let store = {};
 
-const _callsubscribers = (key, newValue, prevValue) => {
-  store[key].subscribers.forEach(callback => {
-    if (typeof callback === 'function') {
-      callback(newValue, prevValue);
+// Thanks to https://stackoverflow.com/a/25456134/1954789
+export function deepEqual(x, y) {
+  // TODO: this will not handle cyclical references !
+  // TODO: this does not check class name !
+
+  if (x === y) {
+    return true;
+  }
+
+  if ((typeof x == "object" && x != null) && (typeof y == "object" && y != null)) {
```

Comment:
Shouldn't we do `===` validations here? ESLint and some static validators will throw some error/warnings here

Reply:
We should provide an eslint config then...